pkg/apic/provisioning/schemabuilder.go

```diff
@@ -53,17 +64,40 @@ func (s *schemaBuilder) SetDescription(description string) SchemaBuilder {
 	return s
 }
 
+// SetPropertyOrder - Set a list of ordered fields to be rendered in the UI
+func (s *schemaBuilder) SetPropertyOrder(propertyOrder []string) SchemaBuilder {
+	// If property names in the property order is bogus, it will be ignored when rendered
+	s.propertyOrder = propertyOrder
+	propertyOrderSet = true
+	return s
+}
+
 // AddProperty - adds a new subscription schema property to the schema
 func (s *schemaBuilder) AddProperty(property PropertyBuilder) SchemaBuilder {
 	prop, err := property.Build()
 	if err == nil {
 		s.properties[prop.Name] = *prop
+		// If property order wasn't set, add property as they come in
```

Comment:
Originally had an order that was weighted.  But now, if property order not set, append them as the property is added.